debezium-platform-conductor/src/main/java/io/debezium/platform/environment/operator/OperatorPipelineController.java

```diff
@@ -236,6 +248,16 @@ public LogReader logReader(Long id) {
         return new KubernetesLogReader(() -> findDeploymentLoggable(id));
     }
 
+    @Override
+    public void sendSignal(Long id, SignalRequest signalRequest) {
+
+        DebeziumServer ds = findById(id)
+                .orElseThrow(() -> new DebeziumException(String.format("Pipeline with id %s not found", id)));
+
+        debeziumServerProxy.sendSignal(signalRequest, ds);
```

Comment:
maybe here we can try to avoid to pass directly the `request` and `map` to a domain object like:

`signalRequest` -> (validation + enrichment) -> `signal`